fs_product_multi_image/views/product_template.xml

```diff
@@ -20,7 +20,11 @@
             </field>
             <page name="sales" position="after">
                 <page name="images" string="Images">
-                    <field name="image_ids">
+                    <field
+                        name="image_ids"
+                        widget="fs_image_relation_dnd_upload"
+                        options="{'target': 'fs_image'}"
+                    >
```

Comment:
IMO, the target should be selected by the user and default on specific. Cases where you want to reuse the uploaded image for others objects are minority. You could for example add a select field into the DND zone on the widget to allows the selection of the target...

Reply:
Yes I thought about this, I will add this.
Also I think it could be interesting to add a dialog where we can select existing fs.image, but this could be done afterwards.

Reply:
I added the select to change the target.